renovatebot/renovate
# docs: use docusaurus for docs generation [WIP]

work in progress

closes #6944 

---
Steps to test:
- yarn install & yarn build
- cd docs/website
- yarn install
- yarn start
- see browser 🙃 

For updating docs use `yarn generate` in second shell in `docs/website` folder. It will live reload browser.
